runtime/src/main/kotlin/kotlin/text/Regex.kt

```diff
@@ -261,21 +261,25 @@ actual public class Regex internal constructor(internal val nativePattern: Patte
      */
     actual fun split(input: CharSequence, limit: Int): List<String> {
```

Comment:
Looks like it isn't important because this default is already specified in the common code and this method can be called in K/N without specifying the second parameter. But providing a default value for our implementation will make the code more readable.